scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -463,7 +523,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
   @tailrec
   private def isAheadNewLine(currentPosition: Int): Boolean = {
     val nextPos = currentPosition + 1
-    if (scannerTokens(currentPosition).is[ColonEol]) true
+    if (isColonEol(scannerTokens(currentPosition))) true
```

Comment:
I think this is no longer needed, as ColonEol was consuming LF, but not it is not so it should be emited correctly

Reply:
Removed! Seems to work now.